### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by whyman10x** [tfutils/tfenv#410]
Seems like some nice stuff has been merged since 3.0, time for a 3.1 to be cut?

**Comment by kvendingoldo** [comment]
Our team built `tenv` (https://github.com/tofuutils/tenv) that is a heir of tfenv/tofuenv. We added support of tofu/terragrunt and migrated codebase to Golang. Welcome to contribute to it.

**Comment by mmiller-max** [comment]
Just bumping this, in particular for me would be nice to get `latest-allowed` released

**Comment by kvendingoldo** [comment]
@mmiller-max try tenv, we added support of `latest-allowed` last month.

**Comment by richid** [comment]
Yep, at this point I have to recommend moving to [tenv](https://github.com/tofuutils/tenv). It's very fast, works well, and supports other tools as well. My only gripe is that the Docker image isn't built for CI, so usage there takes a couple more steps.

**Comment by kvendingoldo** [comment]
@richid Can you share your thoughts about docker image in tenv's issues, please? Our team makes improvements quickly, and we can build an image that will cover your needs. Thanks.

**Comment by richid** [comment]
> @richid Can you share your thoughts about docker image in tenv's issues, please? Our team makes improvements quickly, and we can build an image that will cover your needs. Thanks.

First, apologies. I just realized that the `tfenv` image I had been using in CI was _not_ the official one but an [OSS one purpose built for CI](https://hub.docker.com/r/harisekhon/tfenv). Personally, I really like the ability to run a build using that image and then only have to do 

```
tfenv use
terraform init
terraform plan
...
```

and have everything Just Work™️  . Either way, just my opionion and thank you for the great tool!

**Comment by Zordrak** [comment]
I apologise that the support and release cycle for tfenv is slow.

There are many factors that cause this both personal and professional, however:

1. Rest assured it is still an active project.
2. For any use-case specific issues that have not yet been addressed, merged or released, a quick fork can solve your issues until things are addressed. As tfenv is wholly written in bash, and installable with a clone, it's super easy to run a fork and clone from there instead if you have any impacting issues that you know how to solve. The beauty of bash is that modification and extension is incredibly easy, and you (almost) never need to worry about changes in the language, backwards compatibility or the direction and ownership of the tooling. I actively encourage you to do whatever you want with tfenv. It's an MIT licensed project - get any value you like out of it in any way that works for you.
3. I enjoy that on the whole tfenv is extremely robust, and has had very little change in many years. There are features to add, docs to improve, and the occasional unexpected regex bug, but on the whole, so long as Hashicorp don't change or break their release processes (again), it does what it does quite simply.
4. **Most importantly** if anything happens to actively break tfenv - such as changes to Hashicorp's release/hosting mechanisms - they will be dealt with immediately. tfenv is in heavy active use across many commercially sensitive projects. If it stops working, it will be an immediate concern and will be addressed as urgent.

I can't promise to provide fast and reactive support and development for minor issues and improvements, but tfenv works, and I will do all I can to make sure it stays that way.
